src/GraphQL/Types/Composite/AutoRegisteringInputObjectGraphType.cs

```diff
@@ -76,5 +80,20 @@ protected virtual IEnumerable<PropertyInfo> GetRegisteredProperties()
             => AutoRegisteringHelper.ExcludeProperties(
                 typeof(TSourceType).GetProperties(BindingFlags.Public | BindingFlags.Instance).Where(x => x.CanWrite),
                 _excludedProperties);
+
+        /// <summary>
+        /// Analyzes a property and returns an instance of <see cref="TypeInformation"/>
+        /// containing information necessary to select a graph type. Any <see cref="GraphQLAttribute"/>
```

Comment:
>  Any \<see cref="GraphQLAttribute"/>

Not only those ,
https://github.com/graphql-dotnet/graphql-dotnet/pull/2884/files#r795050710

Reply:
I updated the comment to match.